prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -22,6 +22,7 @@ asap_accepted_issuers = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_ISSU
 asap_accepted_audiences = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_AUDIENCES) }}" }
 {{ end }}
 
+{{ $enable_guest_domain := and $ENABLE_AUTH (.Env.ENABLE_GUESTS | default "0" | toBool)}}
```

Comment:
can you please make the variable name uppercase and move it above with the other variable declarations? LGTM otherwise!

Reply:
I did originally have it uppercase, but then thought lower case would differentiate it from ENV variables as an internal one?

But happy to change it if you can confirm it's what you want given the above.

Thanks